dango/account/safe/src/execute.rs

```diff
@@ -334,33 +361,6 @@ mod tests {
             .with_contract(SAFE)
             .with_mode(AuthMode::Finalize);
 
-        // Someone who is not a member attempts to sender a tx with the Safe.
-        // Should fail.
-        {
-            let res = authenticate(ctx.as_auth(), Tx {
-                sender: SAFE,
-                gas_limit: 1_000_000,
-                // For this test the messages don't matter.
-                msgs: NonEmpty::new_unchecked(vec![]),
-                data: Metadata {
-                    username: non_member.clone(),
-                    // The things below (nonce, chain_id, expiry) don't
-                    // matter, because authentication should fail before we even
-                    // reach the signature verification step.
-                    nonce: 0,
-                    chain_id: chain_id.clone(),
-                    expiry: None,
-                }
-                .to_json_value()
-                .unwrap(),
-                credential: Json::null(),
-            });
-
-            assert!(res.is_err_and(|err| err.to_string().contains(&format!(
-                "account {SAFE} isn't associated with user `{non_member}`"
-            ))));
-        }
```

Comment:
This test case is no longer relevant as the same scenario is already covered by the integration test (in dango-testing)